google/cloud/spanner_dbapi/connection.py

```diff
@@ -37,11 +34,25 @@ class Connection:
     """
 
     def __init__(self, instance, database):
+        self._sessions_pool = BurstyPool()
+        self._sessions_pool.bind(database)
+
         self.instance = instance
         self.database = database
-        self.is_closed = False
 
         self._ddl_statements = []
+        self.transactions = []
+
+        self.is_closed = False
+        self.autocommit = True
```

Comment:
We had a word about it with Chris some time ago, and decided to use a property. It can be found in our meetings notes.

We don't have any mentions of autocommit mode in PEP 249, so it seems like we should ask the Java implementation about it:
https://github.com/googleapis/java-spanner-jdbc/blob/cedd167c5973fe50e0205ae641f6580ebd627884/src/main/java/com/google/cloud/spanner/jdbc/JdbcConnection.java#L81-L82

Thus, there will be both setter and getter of this property